### PR TITLE
Updates4

### PR DESCRIPTION
**Pull request by Tommy1231232**

### CODE REVIEW

File: docs/getting-started/welcome.md [hunk 1]
```diff
@@ -4,39 +4,38 @@ sidebar_label: Welcome to UMA
...
+## Contact Us
 
-For a more detailed view, [API documentation](https://docs-dot-uma-protocol.appspot.com/uma/index.html) is available for all UMA smart contracts.
+If you have any questions or want to connect with the UMA community, please check out our community [page](community/community-overview) or join our [Discord](https://discord.umaproject.org/).
 
-## Contact Us
 
-If you have any questions or want to reach out, please join our [Discord](https://discord.umaproject.org/).
+
+---
```
Comment (smb2796) [inline docs/getting-started/welcome.md:41]: Did you mean to add this bar at the bottom of the page?

**Review by smb2796 — COMMENTED** [review]
Looking good Tom. This section is really coming along. Just left a few comments. Also there appear to be quite a few merge conflicts. Might be from a couple of commits I merged in yesterday. Happy to help resolve if need be.

File: docs/getting-started/oracle.md [hunk 2]
```diff
@@ -12,68 +12,13 @@ Despite a large body of existing research into oracle system design, current app
 Economic guarantees around the cost of corrupting blockchain oracles are critical for the development of useful smart contracts, particularly in decentralized finance (DeFi) applications.
```
[the remark this answers is not in the document]
Comment (Tommy1231232) [reply docs/getting-started/oracle.md:12]: Yep totally agree, my next iteration of this was to try to loop in other benefits to exemplify

File: docs/getting-started/welcome.md (hunk 1 above)
Comment (Tommy1231232) [reply docs/getting-started/welcome.md:41]: No thanks for calling it out. Removed

File: docs/synthetic-tokens/expiring-synthetic-tokens.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
+---
+title: How Expiring Synthetic Tokens Work
+sidebar_label: Expiring Synthetic Tokens
+---
+
+To create a synthetic token, developers use UMA's priceless synthetic contract templates to easily deploy the new contract. To manage and enforce contracts, the contracts leverage UMA's optimistic oracle known as the Data Verification Mechanism (DVM). Below explains how UMA's Expiring MultiParty (EMP) contracts are managed and enforced on UMA.
```
[the remark this answers is not in the document]
Comment (Tommy1231232) [reply docs/synthetic-tokens/expiring-synthetic-tokens.md:6]: I like it as one. I'll update to reflect this

**Pull request closed without merging**